optuna/importance/_fanova/_tree.py

```diff
@@ -255,7 +255,7 @@ def _get_node_children(self, node_index: int) -> Tuple[int, int]:
         return self._get_node_left_child(node_index), self._get_node_right_child(node_index)
 
     def _get_node_value(self, node_index: int) -> float:
-        return self._tree.value[node_index]
+        return float(self._tree.value[node_index])
```

Comment:
I guess we can also extract a float value from `numpy.ndarray` of size 1 with `.item()`, but it can return the other type like `int` depending on the content. So, the current implementation seems reasonable to me. What do you think?

```suggestion
        return self._tree.value[node_index].item()
```

Reply:
Your suggestion depends on the `scikit-learn` implementation, but it looks good to me because it returns a float.

Reply:
I rethink your suggestion and revert the change because it's too implementation dependent, which assumes `self._tree.value`'s dimension is 2 or more.

Reply:
Thank you for your investigation. Sounds reasonable to me.